docs/HTTP3.md

```diff
@@ -27,6 +27,9 @@ notice. It needs to be enabled at build-time.
 Further development and tweaking of the HTTP/3 support in curl will happen in
 in the master branch using pull-requests, just like ordinary changes.
 
+To build on MacOS with provided recipes you may want to set `alias nproc="sysctl -n hw.ncpu"`
+before running make commands.
```

Comment:
I propose we drop the `-j` use all together instead to make the command lines work better on more places.

Reply:
May be we should add a note that parallel builds are supported with a `-j <numcpu>` keyword, so maintainers will use that in their build recipes?

Reply:
P.S. Just checked `sysctl -n hw.ncpu` also working on the FreeBSD, so it seems that with ncpu for the Linux and alias for the *BSD/Darwin we are covering most of the systems.

Reply:
I don't think we need to promote "make tips" here and maintainers should really already know about `make -j` ...